### DISCUSSION TITLE
Disabling lazyloading on columns and rows

### DISCUSSION THREAD

**Discussion by JEDacreWright** [ghiscoding/Angular-Slickgrid#1072] [Q&A]
Hi !

I wish you a happy new year.
I searched for some time now any way to prevent Slickgrid from creating and deleting columns as I scroll. I know I have not much columns displayed and wanted to disable this feature as it seems to cause "stutter" on the scroll.

[Video of the issue]
[Slickgrid-Stutter.webm](https://user-images.githubusercontent.com/24585950/210347241-e965a905-a7c0-4185-89cb-56db69b51b1a.webm)

Is there any way to disable it with existing properties or methods ?
It might be too much to ask, but being able to disable an axis (row vs column) lazyloading independently to the other one would be heaven on earth.

Thank you for the amazing work you've been doing for years now on this library.

**Comment by ghiscoding** [comment]
I don't think that it can be disabled, the Virtual Scroll is a SlickGrid feature that was created by the original author and I don't really know how it works. However there's a [`minRowBuffer`](https://github.com/6pac/SlickGrid/blob/6e4b6b66a05ef1b5cf36ddb604f2a0e81f8d0a3c/slick.grid.js#L105) which you can increase (its default is 3), however I can only assume this is for vertical scroll not for horizontal scroll, I'm not sure that can help with your problem. I don't typically have problems with stuttering but I don't usually have that many columns. I'm afraid I can't be of more help than that. Happy New Year

**Reply by JEDacreWright** [reply]
Thanks for your fast answer. If I have time and find a way, may I give you some code to add such a feature (PR or else) ?

**Reply by ghiscoding** [reply]
PRs are always welcome :)

**Reply by ghiscoding** [reply]
so I've seen similar stuttering on some grids in Chrome/Edge but that doesn't occur in Firefox, so that might be a Chrome bug
